app/views/protips/index.html.haml

```diff
@@ -13,10 +13,10 @@
   .container
     .clearfix
       .sm-col.sm-col.sm-col-12.md-col-8
-        -if first_page?
-          .purple{style: "border-bottom:solid 5px;"}
-            %h2.mt0.black=protips_heading
-            %p.clearfix.py1.font-lg.black=protips_description
+        -# -if first_page?
```

Comment:
This too?

Reply:
Screenshots for before/after would help here too.

Reply:
There was no h1 tag at all pages after the first one.
![Screenshot from 2019-06-05 13-30-53](https://user-images.githubusercontent.com/5982702/58949984-63c1cc00-8796-11e9-9c5e-ac3ddc162801.png)
![Screenshot from 2019-06-05 13-31-45](https://user-images.githubusercontent.com/5982702/58949986-63c1cc00-8796-11e9-85b9-314052e19da3.png)
![Screenshot from 2019-06-05 13-32-21](https://user-images.githubusercontent.com/5982702/58949987-63c1cc00-8796-11e9-91c5-9e1cf0b0c678.png)